pkgs/applications/networking/cluster/helm/plugins/helm-schema.nix

```
@@ -0,0 +1,66 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "helm-schema";
+  version = "2.2.1";
+
+  src = fetchFromGitHub {
+    owner = "losisin";
+    repo = "helm-values-schema-json";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-dttUbP+G2q+IJwoYReSfJVnc/Dix3Bk0H0EkuCuU9Ko=";
+  };
+
+  vendorHash = "sha256-+Idcps9Z/56rvlSYVX8cw4Rbg/2Kt9bzhwtN3clQ3XY=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X 'main.Version=v${finalAttrs.version}'"
+  ];
+
+  postPatch = ''
+    # Remove the install and upgrade hooks
+    sed -i '/^hooks:/,+2 d' plugin.yaml
+
+    substituteInPlace {plugin.yaml,plugin.complete} \
+      --replace-fail '$HELM_PLUGIN_DIR' '${placeholder "out"}/${finalAttrs.pname}/bin'
+  '';
+
+  postInstall = ''
+    install -D plugin.complete --target-directory=$out/${finalAttrs.pname}
+    install --mode=644 plugin.yaml --target-directory=$out/${finalAttrs.pname}
+    mv $out/bin/{helm-values-schema-json,schema}
+    mv $out/bin $out/${finalAttrs.pname}
```

pname is not supposed to be substituted like this, especially not with finalAttrs.

In $out/bin binaries should life and other files should probably be under $out/share/$NAME

```suggestion
    install -D plugin.complete -t $out/share/helm-schema/
    install plugin.yaml -t $out/share/helm-schema/
```
Setting mode is useless as nix always uses 666 or 777.